Solutions/AzureDevOpsAuditing/Hunting Queries/ADOVariableCreatedDeleted.yaml

```diff
@@ -1,6 +1,8 @@
 id: 0fa523de-ce23-49d0-87a4-e890028e1e50
 name: Azure DevOps - Variable Created and Deleted
-description: |
+description : |
+  'The query detects swift additions and removals of variables in build processes, possibly indicating malicious activity, and suggests promoting it to a detection if few such events occur.'
```

Comment:
The query detects additions and removals of variables in build processes in a short span of time, possibly indicating malicious activity. Promoting to a detection if few such events occur.